server/routes/api/v1/pod/create.js

```diff
@@ -14,7 +14,7 @@ module.exports = function (express) {
 
     // Example of data in JSON format
     // pod.add({current_weight: 124, max_weight: 900, last_maintain: 1200}
-    pod.create({current_weight: clientPodPost.current_weight, max_weight: clientPodPost.max_weight, last_maintain: clientPodPost.last_maintain},
+    pod.add({current_weight: clientPodPost.current_weight, max_weight: clientPodPost.max_weight, last_maintain: clientPodPost.last_maintain},
```

Comment:
according to the models its add i did pull master and then changed them according to what the models say i can change it back though